corehq/apps/accounting/views.py

```diff
@@ -338,8 +338,27 @@ def cancel_form(self):
         return CancelForm()
 
     @property
-    def page_context(self):
+    def invoice_context(self):
+        subscriber_domain = self.subscription.subscriber.domain
+
+        invoice_report = InvoiceInterface(self.request)
+        invoice_report.filters.update(subscription__subscriber__domain=subscriber_domain)
```

Comment:
This came out in the scoping I did with Krishna. He said, billing-team usually want to see all the Invoices for a project space, not particularly for the selected subscription. Should we still change this, if so good to ping the fogbugz case http://manage.dimagi.com/default.asp?163274?